app/controllers/api/v1/users_controller.rb

```diff
@@ -3,6 +3,12 @@
 module Api
   module V1
     class UsersController < ApplicationController
+      before_action :authorization, only: [:index]
+
+      def index
+        @users = User.all
```

Comment:
Please check the [discard documentation](https://github.com/jhawthorn/discard) since we are implementing a soft delete into the User model.

Reply:
Listo, lo revisé y lo modifiqué